docs/setup/configuring-rules.md

```diff
@@ -264,4 +264,52 @@ Example:
 # YAML: Set the AZURE_AKS_ENABLED_PLATFORM_LOG_CATEGORIES_LIST configuration option to cluster-autoscaler and AllMetrics categories only. 
 configuration:
   AZURE_AKS_ENABLED_PLATFORM_LOG_CATEGORIES_LIST: ['cluster-autoscaler', 'AllMetrics']
-```
\ No newline at end of file
+```
+
+### Azure Application gateway additional region availability zone list
```

Comment:
Basically repeating the same example as `Azure.AKS.AvailabilityZone`. Perhaps we could condense the exampels into one?

Reply:
Generally I would suggest separate rule configuration, but in this case it probably make sense to consolidate them.

Reply:
No worries @BernieWhite. I've consolidated the AZ rules. Let me know if your fine with it or want to make any changes 🙂.